Makefile

```diff
@@ -225,7 +235,12 @@ define SHOW_ARCH
   $(shell printf "\\t%s%s\\\n" "$(1)" $(if $(filter $(ARCH),$(1))," (default)",""))
 endef
 
-all: runtime
+all: runtime netmon
+
+netmon: $(NETMON_TARGET_OUTPUT)
+
+$(NETMON_TARGET_OUTPUT): $(SOURCES)
+	$(QUIET_BUILD)(cd $(NETMON_DIR) && go build -i -o $@ -ldflags "-X main.version=$(VERSION)")
```

Comment:
No install target?

Reply:
Yes I added it later in the commits, but I fixed it by rebasing and bringing the change directly to this commit.

Reply:
Please could you:

- Update `show-summary` so that "binaries to install" also iterates through `$(BINLIBEXECLIST)`.
- Update `show-usage` to include `install-bin-libexec` is listed under "Additional targets".
- Potentially also update the `install` target so that it forces `install-bin-libexec` to be called (`sudo make install` should DTRT I think :)

Reply:
- I have updated `show-summary`
- I have updated `install` target so that it invokes `install-bin-libexec` also
- I have __not__ updated `show-usage` since any other subcommand such as `install-bin` was already listed. And also, because now `install` takes care of `install-bin-libexec`, I think that's fine.